csharp/src/Google.Protobuf/JsonFormatter.cs

```diff
@@ -149,6 +149,19 @@ public string Format(IMessage message)
         }
 
         /// <summary>
+        /// Formats the specified field of the message as JSON.
```

Comment:
Given that (assuming I've read the commits properly) you've changed your mind about what this is going to return (one commit returning the field name and value, the other commit only returning the field value) I think it would be worth documenting this.